lib/wasm/WasmModuleTemplatePlugin.js

```diff
@@ -40,50 +40,11 @@ class WasmModuleTemplatePlugin {
 							return `${module.moduleArgument}.exports = instance.exports;`;
 						}
 					};
-					const generateImports = () => {
-						const depsByRequest = new Map();
-						for (const dep of module.dependencies) {
-							if (dep instanceof WebAssemblyImportDependency) {
-								const request = dep.request;
-								let array = depsByRequest.get(request);
-								if (!array) {
-									depsByRequest.set(request, (array = []));
-								}
-								const exportName = dep.name;
-								const usedName = dep.module && dep.module.isUsed(exportName);
-								array.push({
-									exportName,
-									usedName,
-									module: dep.module
-								});
-							}
-						}
-						const importsCode = [];
-						for (const pair of depsByRequest) {
-							const properties = [];
-							for (const data of pair[1]) {
-								properties.push(
-									`\n\t\t${JSON.stringify(
-										data.exportName
-									)}: __webpack_require__(${JSON.stringify(
-										data.module.id
-									)})[${JSON.stringify(data.usedName)}]`
-								);
-							}
-							importsCode.push(
-								`\n\t${JSON.stringify(pair[0])}: {${properties.join(",")}\n\t}`
-							);
-						}
-						return importsCode.join(",");
-					};
 					const source = new RawSource(
 						[
 							'"use strict";',
-							"",
 							"// Instantiate WebAssembly module",
-							"var instance = new WebAssembly.Instance(__webpack_require__.w[module.i], {" +
-								generateImports(),
-							"});",
+							"var instance = __webpack_require__.w[module.i]",
```

Comment:
I made the choose that `__webpack_require__.w` only contains the `WebAssembly.instance` (`WebAssembly.Module` is not preserve because not used)